package.json

```
@@ -2,36 +2,38 @@
   "name": "iconoir",
   "version": "6.11.0",
   "description": "The biggest open source icon library with tons of free icons.",
```

@lucaburgio Should we align the description with the one from the README? Something like

```suggestion
  "description": "An open-source library with 1300+ unique SVG icons.",
```